public_html/wp-content/mu-plugins/blocks/source/blocks/speakers/block-content.js

```diff
@@ -35,63 +33,59 @@ import './block-content.scss';
 function SpeakerSessions( { speaker, tracks } ) {
 	const sessions = get( speaker, '_embedded.sessions', [] );
 
-	let output = ( <Fragment>{ null }</Fragment> );
-
-	if ( sessions.length ) {
-		output = (
-			<div className="wordcamp-speakers__sessions">
-				<h4 className="wordcamp-speakers__sessions-heading">
-					{ _n( 'Session', 'Sessions', sessions.length, 'wordcamporg' ) }
-				</h4>
-
-				<ul className="wordcamp-speakers__sessions-list">
-					{ sessions.map( ( session ) =>
-						<li
-							key={ session.id }
-							className="wordcamp-speakers__sessions-list-item"
-						>
-							<Disabled>
```

Comment:
Why was `<Disabled>` removed? it'd also be good to document the reason in the commit message.

Reply:
For the same reason it's been removed in other PRs - it doesn't play well with links. I didn't think to explain it again, but I can add it in the merge commit if you think it's necessary.

Reply:
More discussion about the removal here:
https://github.com/WordPress/wordcamp.org/pull/171#issuecomment-516200569

I think it's probably worth adding to the merge commit message.

Reply:
Ah, I'd forgotten about that, thanks for the reminder :)